executor/src/main.rs

```diff
@@ -0,0 +1,116 @@
+use std::{io::Write, process::Stdio};
+
+use tempfile::NamedTempFile;
+use uuid::Uuid;
+
+/// The Playwright template with a placeholder for the script
+const PLAYWRIGHT_TEMPLATE: &str = r#"
+const { chromium } = require('playwright-core');
+const { prompt, prove, setSessionUUID } = require("@plutoxyz/playwright-utils");
+
+(async () => {
+  const sessionUUID = process.argv[2];
+  setSessionUUID(sessionUUID);
+  console.log("Starting Playwright session with UUID:", sessionUUID);
+
+  const browser = await chromium.launch({
+    headless: true,
```

Comment:
let's make this config object configurable by rust, so maybe we place a json serialized object here?